src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="search_input_big form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Attributes with long values or having more than 2 attributes should be placed on separate lines with proper indentation. The `class` attribute in the `input` element (line 25) should be on a new line with a 2-space indentation.